oqupy/bath_correlations.py

```diff
@@ -362,6 +386,10 @@ class CustomSD(BaseCorrelations):
         ``'gaussian'``}
     temperature: float
         The environment's temperature.
+    alt_integrator: bool
```

Comment:
I think in docstrings other than CustomSD (or whatever you deem the 'parent' one to be) we can either a) copy the entire docstring or b) (probably neater) reference CustomSD, e.g.

Whether to use an alternative integration scheme leveraging sine/cosine weighted versions of scipy.integrate.quad to handle rapid oscillations. See...